nipo/templates/404.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'base,html' %}
+
+
+{% block title %}
+page not found
+{% endblock %}
+
+
+{% block content %}
+|<h1>404 error</h1>
+<p>page Not found,Try to use our search form</p>
```

Comment:
What is the search form? No such thing exists on the site. I would suggest you just leave it as page not found